modules/oe_translation_corporate_workflow/modules/oe_translation_active_revision/tests/src/FunctionalJavascript/ActiveRevisionTest.php

```diff
@@ -1483,6 +1477,324 @@ public function testLimitingVersionsToMap(): void {
+    $this->getSession()->getPage()->pressButton('Save and synchronise');
+    $this->assertSession()->pageTextContains('The translation has been saved.');
+    $this->assertSession()->pageTextContains('The translation has been synchronised.');
+    $this->drupalGet('/node/' . $node->id() . '/translations');
+    $this->assertEquals('Version 2.0.0', $french_row->find('xpath', '//td[2]')->getText());
+    $this->assertEquals('Mapped to version 1.0.0', $italian_row->find('xpath', '//td[2]')->getText());
+    $active_revision_storage->resetCache();
+    $active_revision = $active_revision_storage->load($active_revision->id());
+    $language_values = $active_revision->get('field_language_revision')->getValue();
+    $this->assertCount(1, $language_values);
+    // Only the IT mapping remained.
+    $this->assertEquals([
+      'entity_type' => 'node',
+      'entity_id' => $node->id(),
+      'entity_revision_id' => $version_one_revision_id,
+      'langcode' => 'it',
+      'scope' => 0,
+    ], $language_values[0]);
+
+    // Create a new translation of version 2 in IT, using the remote
+    // translation.
+    $this->drupalGet('/node/' . $node->id() . '/translations/remote');
+    $this->getSession()->getPage()->selectFieldOption('Translator', 'remote_one');
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->getSession()->getPage()->checkField('Italian');
+    $this->getSession()->getPage()->pressButton('Save and send');
+    $this->assertSession()->pageTextContains('The translation request has been sent');
+    $requests = \Drupal::service('plugin.manager.oe_translation_remote.remote_translation_provider_manager')->getExistingTranslationRequests($node, FALSE);
+    $this->assertCount(1, $requests);
+    $request = reset($requests);
+    TestRemoteTranslationMockHelper::translateRequest($request, 'it');
+    $request->save();
+    $this->getSession()->reload();
+    $this->getSession()->getPage()->clickLink('Review');
+    $this->getSession()->getPage()->pressButton('Save and synchronise');
+    $this->assertSession()->pageTextContains('The translation in Italian has been synchronised.');
+
+    // The final mapping has been removed and we no longer have an active
+    // revision entity (as it was deleted due to not having any mappings left).
+    $this->drupalGet('/node/' . $node->id() . '/translations');
+    $table = $this->getSession()->getPage()->find('css', 'table.existing-translations-table');
+    $french_row = $table->find('xpath', '//tr[@hreflang="fr"]');
+    $italian_row = $table->find('xpath', '//tr[@hreflang="it"]');
+    $this->assertEquals('Version 2.0.0', $french_row->find('xpath', '//td[2]')->getText());
+    $this->assertEquals('Version 2.0.0', $italian_row->find('xpath', '//td[2]')->getText());
+    $active_revision_storage->resetCache();
+    $this->assertCount(0, $active_revision_storage->loadMultiple());
+  }
+
+  /**
+   * Tests that when we sync a translation, the mapping is removed.
```

Comment:
The comment is similar to the previous test method.

Reply:
Yes, because it tests the same thing but under a different scenario. It's explained in the next line.